app/dashboard/src/components/JSONSchemaInput.tsx

```diff
@@ -92,7 +92,7 @@ export default function JSONSchemaInput(props: JSONSchemaInputProps) {
                   value={typeof value === 'string' ? value : ''}
                   size={1}
                   className={twMerge(
-                    'focus-child text w-60 grow rounded-input border-0.5 bg-transparent px-input-x read-only:read-only',
+                    'focus-child text w-60 w-full grow rounded-input border-0.5 bg-transparent px-input-x read-only:read-only',
```

Comment:
also with the changes in this file i guess make sure nested objects (nested divs) still look fine

Reply:
I tested the datalink dialog and everything seem to me correct

Reply:
nope! if everything looks fine there then it's all good